plugins/inputs/supervisor/README.md

```diff
@@ -0,0 +1,88 @@
+# Supervisor Input Plugin
+
+This plugin gather information about processes that running under supervisor using XML-RPC API
+
+Plugin minimum tested version: 3.3.2
+
+### Supervisor configuration
+
+This plugin needs TCP HTTP server to be enabled for collecting information. Here is example of
+`inet_http_server` section in supervisor config that will work with default plugin configuration.
+
+```
+[inet_http_server]
+port=127.0.0.1:9001
+```
+
+
+
+### Plugin configuration
+
+```toml
+[inputs.supervisor]
+  ## Url of supervisor's XML-RPC endpoint if basic auth enabled in supervisor http server,
+  ## than you have to add credentials to url (ex. http://login:pass@localhost:9001/RPC2)
+  # url="http://localhost:9001/RPC2"
+  ## Use supervisor identification string as server tag
+  use_identification_tag = false
+  ## Gather PID of running processes
+  gather_pid = false
+  ## Gather exit codes of processes
+  gather_exit_code = false
```

Comment:
I suggest to do
```suggestion
  ## Select the features to gather
  ## Possible options are "pid" and "exit code". If the include list is empty all features are selected.
  ## Please note: If a feature is both specified in include and exclude, include will take precedence.
  # feature_includes = []
  # feature_excludes = ["pid", "exit code"]
```
to be future proof. This way we can easily add more options without modifying the structure.

Reply:
I'll try to implement this later this week